script/Arcueid/Day 10/10_00_ARC10_5D.txt

```diff
@@ -814,12 +833,13 @@ It's only Arcueid who's trembling violently.
 [sha:0f82be9782ef9b1d007d8a92570d1d5c6e572c3b]{
 -- Page 609, Offset 25871.
 -- 苦しげな呼吸。
-An agonizing breath.
+Her breathing is pained.
 }
 [sha:137b914d38d6306820187f074d5e659675606d4d]{
 -- Page 609, Offset 25872.
 -- アルクェイドの息遣いはこんなにも近いのに、遥か遠くのものに感じる。
-Even though her breath is so close, it feels so far away.
+// Mild liberties to avoid a dumb sounding repetition of breathing
+Even though each breath is so close, they feel so far away.
```

Comment:
is so close, it feels so far away

Reply:
Hm I think plural is correct here? There are multiple breaths

Reply:
Although I'm not so sure I agree with my reasoning anymore, I do agree with my assessment that something was off. I say reallocate the "each" to the second bit.

> Even though they're so close, each breath feels so far away.